### PR TITLE
BL-1728: Remove SMS feature.

### PR DESCRIPTION
**Pull request by dkinzer**
Disable and remove code assoc. to sms feature.

### CODE REVIEW

File: app/assets/stylesheets/partials/_email_form.scss [hunk 1]
```diff
@@ -1,26 +1,10 @@
 @import "_vars.scss";
 
-/* === TEXT MESSAGE/EMAILS  === */
+/* === /EMAILS  === */
```
Comment (ebtoner) [inline app/assets/stylesheets/partials/_email_form.scss:3]: I think that we can delete this entire file. While this header says "TEXT MESSAGE/EMAILS", the styles really only apply to the SMS form.

Comment (dkinzer) [reply app/assets/stylesheets/partials/_email_form.scss:3]: OK, I was worried parts of it were still being used.

Comment (dkinzer) [reply app/assets/stylesheets/partials/_email_form.scss:3]: Specifically the bits that I didn't remove.

Comment (ebtoner) [reply app/assets/stylesheets/partials/_email_form.scss]: I'll doublecheck the email modal on QA once merged, but I looked over the classes/ids and am fairly confident that everything can go.

Comment (dkinzer) [reply app/assets/stylesheets/partials/_email_form.scss]: Oh, OK.  Let me know.  In the mean time I did go ahead a remove the entire file.

**Review by ebtoner — APPROVED** [review]

**Pull request merged**